src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -1532,6 +1532,7 @@ public object ConvertBack(object value, Type targetType, object parameter, Syste
             {
```

Comment:
Please do a `git checkout master src/DynamoCoreWpf/UI/Converters.cs` to revert this file, it is unnecessarily polluting the PR.